torch/quantization/fx/_equalize.py

```diff
@@ -280,12 +285,30 @@ def maybe_get_next_input_eq_obs(node: Node, modules: Dict[str, nn.Module]) -> Op
     However, if there are no connecting layers:
         x -> inp_obs1 -> eq_obs1 -> linear1 -> out_obs1 -> add
     Then we want to return None.
+
+    In the case of an unfused linear-relu layer with a connecting linear layer:
+        linear1 -> relu -> out_obs1 -> eq_obs2 -> linear2 -> out_obs2
+    Since it is unfused, we want to skip over the relu layer and return eq_obs2,
+    the following equalization observer for linear2.
     """
 
     assert(node_supports_equalization(node, modules))
 
-    # Locate the following output observer if it exists
-    maybe_obs_node = maybe_get_next_module(node, modules, ObserverBase)
+    # Locate the following nn.ReLU or F.relu node if it exists
+    maybe_relu_node = maybe_get_next_module(node, modules, nn.ReLU)
+    if maybe_relu_node is None:
+        for user, _ in node.users.items():
+            if user.op == 'call_function' and user.target == F.relu:
+                maybe_relu_node = user
```

Comment:
What does this code do? Can this be covered in `maybe_get_next_module` call?

Reply:
This part of the code is similar to what `maybe_get_next_module` call is doing, except we're looking for an `F.relu` layer. I'm not sure how to check if a layer is `F.relu` using `maybe_get_next_module` because currently the way I check the module type is different for functional layers. Instead of doing `isinstance(modules[user.target], nn.relu)`, I just check `user.target == F.relu`. 

Ideally I want to edit `maybe_get_next_module` to be something like the following, and call it by doing `maybe_get_next_module(node, modules, F.relu)`, but this results in an error because `isinstance()` requires a type.
```python
def maybe_get_next_module(node: Node, modules: Dict[str, nn.Module], target_module_type: Any,) -> Optional[Node]:
    for user, _ in node.users.items():
        if user.op == 'call_module' and isinstance(modules[str(user.target)], target_module_type):
            return user
        elif user.op == 'call_function' and user.target == target_module_type:
            return user

    return None
```

If there is a better way of checking if a node is `F.relu` that could allow us to extend this function, that would be greatly appreciated!